src/geop/processing/src/main/scala/com/azavea/mmw/TR55.scala

```diff
@@ -0,0 +1,395 @@
+    2479778 -> "A",
+    2479779 -> "A",
+    2479781 -> "A",
+    2479782 -> "A",
+    2479783 -> "A",
+    2479784 -> "A",
+    2479785 -> "A",
+    2479786 -> "A",
+    2479787 -> "B",
+    2479788 -> "A/D",
+    2479789 -> "B",
+    2479790 -> "B",
+    2479791 -> "B",
+    2479792 -> "B",
+    2479793 -> "B",
+    2479794 -> "B",
+    2479795 -> "B",
+    2479796 -> "B",
+    2479797 -> "B",
+    2479798 -> "B",
+    2479799 -> "B",
+    2479800 -> "C",
+    2479801 -> "C",
+    2479802 -> "C",
+    2479803 -> "C",
+    2479804 -> "B/D",
+    2479805 -> "B/D",
+    2479806 -> "B/D",
+    2479807 -> "B/D",
+    2479808 -> "C/D",
+    2479809 -> "B",
+    2479810 -> "B",
+    2479811 -> "C/D",
+    2479812 -> "D",
+    2479813 -> "D",
+    2479814 -> "D",
+    2479815 -> "D",
+    2479816 -> "D",
+    2479817 -> "C",
+    2479818 -> "C",
+    2479819 -> "B",
+    2479820 -> "B",
+    2479821 -> "B",
+    2479822 -> "B",
+    2479835 -> "C/D",
+    2479836 -> "C/D",
+    2479837 -> "C",
+    2479838 -> "C",
+    2479839 -> "C",
+    2479840 -> "C",
+    2479841 -> "C/D",
+    2479842 -> "C",
+    2479843 -> "C",
+    2479844 -> "C",
+    2479845 -> "C",
+    2479846 -> "C/D",
+    2479847 -> "C",
+    2479848 -> "C",
+    2479849 -> "C",
+    2479850 -> "C",
+    2479851 -> "C",
+    2479852 -> "C",
+    2479853 -> "B/D",
+    2479854 -> "B/D",
+    2479855 -> "C",
+    2479856 -> "B",
+    2479857 -> "B",
+    2479858 -> "B",
+    2479859 -> "C",
+    2479860 -> "C",
+    2479861 -> "C",
+    2479862 -> "B",
+    2479865 -> "B/D",
+    2479866 -> "D",
+    2479867 -> "D",
+    2479868 -> "D",
+    2479869 -> "B",
+    2479870 -> "B",
+    2479871 -> "C/D",
+    2479872 -> "C/D",
+    2479873 -> "B",
+    2479874 -> "B",
+    2479875 -> "B",
+    2479876 -> "B",
+    2479877 -> "B",
+    2479878 -> "C/D",
+    2479879 -> "B",
+    2479880 -> "B",
+    2479881 -> "B",
+    2479882 -> "C",
+    2479883 -> "C",
+    2479884 -> "C",
+    2479885 -> "B",
+    2479886 -> "B",
+    2479887 -> "B",
+    2479888 -> "B",
+    2479889 -> "B",
+    2479890 -> "B",
+    2479891 -> "B",
+    2479892 -> "B",
+    2479893 -> "B",
+    2479894 -> "B",
+    2479895 -> "B",
+    2479896 -> "B",
+    2479897 -> "C/D",
+    2479898 -> "C/D",
+    2479899 -> "C/D",
+    2479900 -> "B",
+    2479901 -> "B")
+
+  /** Returns a map of values to counts */
+  def combinations(nlcd: RasterRDD[SpatialKey], soil: RasterRDD[SpatialKey], polygon: Polygon): Map[(Int, Option[String]), Int] = {
+    val mapTransform = nlcd.metaData.mapTransform
+    val nlcdAndSoil = nlcd.join(soil)
+    val histogram = nlcdAndSoil
+      .map { case (key, (nlcdTile, soilTile)) =>
+        val extent = mapTransform(key) // transform spatial key to map extent
+        val rasterExtent = RasterExtent(extent, nlcdTile.cols, nlcdTile.rows) // transform extent to raster extent
+        val clippedPolygon = polygon & extent
+        val localHistogram = mutable.Map[(Int, Option[String]), Int]()
```

Comment:
Seems like calling this a histogram is a misnomer, since we have a `Histogram` type and this is definitely not that.